packages/js/src/ui/components/Popover.tsx

```diff
@@ -0,0 +1,88 @@
+import { Accessor, createContext, createSignal, JSX, Setter, useContext } from 'solid-js';
+import { useUncontrolledState } from '../helpers';
+import { PopoverContent } from './PopoverContent';
+import { PopoverTarget } from './PopoverTarget';
+
+type Direction = 'top' | 'bottom' | 'left' | 'right';
+type AnchorPosition = 'start' | 'end';
+type Position = Direction | `${Direction}-${AnchorPosition}`;
+type Offset = {
+  x: number | string;
+  y: number | string;
+};
+
+export type PopoverProps = {
+  opened?: boolean;
```

Comment:
✏️ Isn't `open` the norm here?

Reply:
I've seen libraries using both, let's go with `open` for now, it's shorter.